### PR TITLE
revised alerts and spinner styling

### PR DESCRIPTION
**Pull request by benjiwheeler**
### Resolves

- Resolves https://github.com/LLK/scratch-gui/issues/3686 and ttps://github.com/LLK/scratch-gui/issues/3941

### Proposed Changes

Various changes to alert styling:
* font size bigger
* better, bigger padding within alerts
* alert width adjusts naturally from 200px to 448px as content demands
* text aligned to left, not center; everything aligns to vertical middle
* spinner color from white to green

### Reason for Changes

_Explain why these changes should be made_

### CODE REVIEW

**Comment by benjiwheeler** [comment]
OK, improved a bunch of things -- many will only show up in standalone (aka playground) mode, but others will help the alert adapt to various languages (and their varying string lengths...)

I ran this design by @kathymakes.

Here's how this alert looks now, with content strings of various length:

image

image

image

...also, fixed the spinner issue.

**Comment by paulkaplan** [comment]
Great thanks for making those updates!

File: src/components/gui/gui.css [hunk 1]
```diff
@@ -303,13 +303,20 @@ $fade-out-distance: 15px;
 /* Alerts */
 
 .alerts-container {
-    width: 448px;
+    /* width: 448px;
```
Comment (paulkaplan) [inline src/components/gui/gui.css]: Can we remove this code?

**Comment by benjiwheeler** [comment]
Removed commented code

**Pull request merged**